src/components/LenderProfile/LenderSummary.vue

```diff
@@ -53,20 +60,97 @@
 				</div>
 			</div>
 		</div>
+		<kv-lightbox
+			:visible="lightboxVisible"
+			@lightbox-closed="lightboxClosed"
+			:title="lightboxTitle"
+		>
+			<!-- eslint-disable max-len vue/singleline-html-element-content-newline -->
+			<template v-if="!isLoggedIn">
+				<p>
+					You must be signed in and have made loans on Kiva first to send a Lender Message. This helps us reduce the amount of SPAM and unwanted messages in the system.
+				</p>
+			</template>
+
+			<template v-if="isLoggedIn && loansNumber == 0">
+				<p>In order to send a message, you will need to make a loan using your own funds first. This helps us prevent spam and unwanted messages.</p>
+				<p class="tw-mt-2">
+					Please
+					<router-link to="/lend">
+						make a loan first
+					</router-link>
+					(note that loans made with bonus credit are not eligible) and then try your message again!
+				</p>
+			</template>
+
+			<template v-if="isLoggedIn && loansNumber > 0">
+				<div>
+					<p
+						v-if="errorMessage"
+						class="tw-text-danger tw-text-small tw-mb-1.5"
+					>
+						{{ errorMessage }}
+					</p>
+					<div class="tw-relative">
+						<textarea
+							class="
+								tw-w-full tw-border tw-border-tertiary tw-rounded-sm tw-px-2 tw-py-1 tw-min-h-16 tw-mb-1.5
+								tw-ring-inset tw-appearance-none tw-resize-none
+								focus:tw-outline-none focus:tw-ring-2 focus:tw-ring-action focus:tw-border-transparent
+							"
+							v-model="lenderMessage"
+						>
+					</textarea>
+						<kv-material-icon
+							class="tw-w-2.5 tw-h-2.5 tw-absolute tw-bottom-3 tw-right-1"
+							:icon="mdiPencilOutline"
+						/>
+					</div>
+					<div class="tw-flex tw-flex-col lg:tw-flex-row tw-gap-2">
+						<kv-button
+							class="tw-w-full lg:tw-w-auto"
+							variant="secondary"
+							:state="sendButtonState"
+							@click="sendMessage"
+						>
+							Send message
+						</kv-button>
+						<kv-button
+							class="tw-w-full lg:tw-w-auto"
+							variant="secondary"
+							@click="closeLightbox"
+						>
+							Close
+						</kv-button>
+					</div>
+				</div>
+				<p class="tw-mt-4">
+					Lender messages shouldn't be used for self-promotion, advertising or solicitation. More information can be found in
+					<router-link to="/kiva-community-guidelines">Kiva's community guidelines</router-link>.
```

Comment:
Analytics?

Reply:
We have other ticket for all analytics events on the page: https://kiva.atlassian.net/browse/MP-570